xscen/aggregate.py

```diff
@@ -689,16 +693,20 @@ def produce_horizon(
     Parameters
     ----------
     ds: xr.Dataset
-      Input dataset with a time dimension.
+        Input dataset with a time dimension.
     indicators:  Union[str, Path, Sequence[Indicator], Sequence[Tuple[str, Indicator]]]
-      Indicators to compute. It will be passed to the `indicators` argument of `xs.compute_indicators`.
-    period: list
-      [start_year, end_year] of the period to be evaluated.
-      If None, the whole time coordinate is used.
+        Indicators to compute. It will be passed to the `indicators` argument of `xs.compute_indicators`.
+    periods: list
+        List of [start_year, end_year] for the period(s) to be evaluated.
```

Comment:
just [start_year, end_year] should also be allowed (rather than [[start_year, end_year]]). It is allowed elsewhere,https://github.com/Ouranosinc/xscen/blob/bf1c9a4cdc4c1d68aed199ff72844b822fce37e8/xscen/extract.py#L108